Ignoring Flask: markers 'python_version <= "3.8"' don't match your environment
Ignoring Flask-Security-Too: markers 'python_version <= "3.9"' don't match your environment
Ignoring WTForms: markers 'python_version <= "3.9"' don't match your environment
Ignoring qrcode: markers 'python_version <= "3.8"' don't match your environment
Ignoring Werkzeug: markers 'python_version <= "3.8"' don't match your environment
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x0000013CF2024AA0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/flask/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x0000013CF2025220>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/flask/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x0000013CF20253A0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/flask/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x0000013CF20255E0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/flask/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x0000013CF20257F0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/flask/
ERROR: Could not find a version that satisfies the requirement Flask==3.1.* (from versions: none)
ERROR: No matching distribution found for Flask==3.1.*
